spec/preloading_spec.cr

```diff
@@ -122,7 +124,8 @@ describe "Preloading" do
   it "preloads has_many through" do
     with_lazy_load(enabled: false) do
       tag = TagBox.create
-      _unused_tag = TagBox.create
+      # ameba:disable Lint/UselessAssign
+      unused_tag = TagBox.create
```

Comment:
Is there a way to tell ameba to ignore variables starting with `_`? That's how we name variables that aren't used but are there just to clarify what they do.

If we can't do that then I think it'd be better to add a comment: `# unused tag` instead of disabling the rule

Reply:
For now Ameba does not ignore variables with `_`, so I had to use `# ameba:disable Lint/UselessAssign` comment.